From: Harald Hoyer <harald@redhat.com>
Date: Wed, 3 Sep 2014 13:28:31 +0200
Subject: initrd-parse-etc.service: ignore return code of daemon-reload

It seems the return code of systemctl daemon-reload can be !=0 in some
circumstances, which causes a failure of the unit and breaks booting in
the initrd.

(cherry picked from commit f3b8fbb1da6519e14985ea444f8304673d20ad3f)
---
 units/initrd-parse-etc.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/initrd-parse-etc.service.in b/units/initrd-parse-etc.service.in
index c0b2543..42c059b 100644
--- a/units/initrd-parse-etc.service.in
+++ b/units/initrd-parse-etc.service.in
@@ -16,7 +16,7 @@ ConditionPathExists=/etc/initrd-release
 
 [Service]
 Type=oneshot
-ExecStartPre=@rootbindir@/systemctl daemon-reload
+ExecStartPre=-@rootbindir@/systemctl daemon-reload
 # we have to retrigger initrd-fs.target after daemon-reload
 ExecStart=-@rootbindir@/systemctl --no-block start initrd-fs.target
 ExecStart=@rootbindir@/systemctl --no-block start initrd-cleanup.service
